neonvm/config/crd/bases/vm.neon.tech_virtualmachines.yaml

```diff
@@ -2521,6 +2521,18 @@ spec:
                 additionalProperties:
                   type: string
                 type: object
+              overcommitFactor:
+                description: "OvercommitFactor sets a factor by which to discount
+                  the resource usage from this VM. For example, if all VMs have an
+                  OvercommitFactor of 2.0, then twice as many VMs of the same size
+                  could be scheduled onto nodes of the same size. \n OvercommitFactor
+                  is INHERENTLY DANGEROUS. Setting this value greater than 1 ONLY
+                  makes sense when all your VMs are much smaller than the nodes they
```

Comment:
General idea:
If VMs are not sufficiently small relative to the size of a node, then there's increased risk from VMs suddenly using more of their resources. If all the VMs are super small, then this is less likely because you'd need more of them to do the same thing at the same time.